### PR TITLE
feat:implement functionality related to Post

### PR DESCRIPTION
**Pull request by mariyasada**
implementing authentication and post related functionality
-user can add,delete,edit note
-changing  backend from Mockbee to firebase

### CODE REVIEW

**Review by tushartiwari7 — APPROVED** [review]
Firebase integration is working fine!
Goodtomerge.

File: src/components/usersPost/UsersPost.jsx [hunk 1]
```diff
@@ -13,58 +13,48 @@ import {
...
-  } = usePosts();
+export const UsersPost = ({ Post, setPostData, setIsEditing }) => {
+  const { user } = useSelector((store) => store.auth);
   const { pathname } = useLocation();
   const [isShow, setShow] = useState(false);
+  const dispatch = useDispatch();
 
-  const seteditData = (post) => {
+  // console.log(Post);
+  // console.log(setIsEditing);
```
Comment (Bhakti199) [inline src/components/usersPost/UsersPost.jsx]: remove comments

File: src/index.js [hunk 2]
```diff
@@ -2,19 +2,24 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
-import { makeServer } from "./server";
+// import { makeServer } from "./server";
 import {BrowserRouter as Router } from "react-router-dom";
 import { CombineProvider } from "./contexts/combineProvider";
+import { Provider } from "react-redux";
+import { store } from "./redux/store";
+console.log(store.getState());
```
Comment (Bhakti199) [inline src/index.js:10]: remove unwanted `consol.log`

**Review by Bhakti199 — APPROVED** [review]
- This code looks fine to me.
- Modular code.
- heirarchy is maintained.
- Good to see that you are using firebase integration.
- everything looks fine to me.
- this pr is good to merge.

**Pull request merged**